index.html

```diff
@@ -28,16 +28,16 @@
 
         <section class="headline">
             <h1 class="intro">Hey there. I'm Ahmad</h1>
-            <h2 class="profession">I'm a software developer.</h2>
+            <h2 class="profession">I'm a software developer</h2>
 
             <p>I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
 
             <ul>
-                <li><a href=#"><img src="./images/twitter.png"></a></li>
-                <li><a href=#"><img src="./images/in.png"></a></li>
-                <li><a href=#"><img src="./images/M.png"></a></li>
-                <li><a href=#"><img src="./images/github.png"></a></li>
-                <li><a href=#"><img src="./images/V.png"></a></li>
+                <li id="twitter"><a href=#"><img src="./images/twitter.png"></a></li>
+                <li id="linkedin"><a href=#"><img src="./images/in.png"></a></li>
+                <li id="m"><a href=#"><img src="./images/M.png"></a></li>
+                <li id="github"><a href=#"><img src="./images/github.png"></a></li>
+                <li id="v"><a href=#"><img src="./images/V.png"></a></li>
             </ul>
             <i class="fa-solid fa-chevron-down"></i>
         </section>
```

Comment:
- [ ] [REQUIRED] The goal of the project milestone is to make your portfolio website **_RESPONSIVE_**, meaning that 
- Desktop view starts from 768, 769px to 1440px and above. You did well in styling the desktop view like the Figma design but the contents still need to be centered or aligned properly as close as possible to the Figma design.
- For screen sizes between the `mobile view - 375px` and `desktop view - 768px` should continue to display like a mobile view. This is were you have things to do to improve your project. 

Your project at a screen size of 686px 👇🏾 - this should still display like the mobile view. 
![image](https://user-images.githubusercontent.com/84156714/220879525-f8e128ea-4d05-40e4-b95e-b9b2f43b5ff8.png)

Your project at a screen size of `767px` 👇🏾 This should still display like the mobile view. 
![image](https://user-images.githubusercontent.com/84156714/220881529-a67ccb85-70f8-42bb-a852-7f2f07023f4e.png)

Your project at screen-size of 768px 👇🏾 - Desktop view begins.
![image](https://user-images.githubusercontent.com/84156714/220882085-8c1a6543-28fc-4331-9b7e-2a4c9a4625fe.png)

It should display like the Figma Desktop template. All its content should be properly aligned.

Reply:
All such overflows were fixed in the newer version of stylesheet by introducing some more break points in between